Source/dictionaries/original.json

```diff
@@ -13,43 +14,56 @@
 		"Can't Handle" : "Can Totally Handle",
 		"cannot even handle" : "Can Probably Totally Handle",
 		"Cannot Even Handle" : "Can Probably Totally Handle",
+        "Debate" : "Dance-off",
 		"Doesn't want you to see" : "Doesn't Really Care If You See",
-		"epic" : "mundane",
+        "Drone": "Dog",
 		"Epic" : "Mundane",
+        "Expands" : "Physically Expands",
+        "First Degree" : "Friggin' Awful",
+        "First-Degree" : "Friggin' Awful",
+        "Front Runner" : "Blade Runner",
+        "Global" : "Spherical",
 		"Go Viral" : "Be Overused So Much That You'll Silently Pray for the Sweet Release of Death to Make it Stop",
-		"greatest" : "average",
 		"Greatest" : "Average",
+        "Horsepower" : "Tons of Horsemeat",
 		"Incredible" : "Painfully Ordinary",
 		"Infuriate" : "Mildly Annoy",
 		"literally" : "figuratively",
 		"Literally" : "Figuratively",
 		"Mind Blowing" : "Mind-Numbingly Ordinary",
 		"Mind-Blowing" : "Painfully Ordinary",
+        "No Indication" : "Lots of Signs",
 		"Nothing Could Prepare Me For" : "Does ANYONE Fucking Care About",
-		"of all time" : "for now",
 		"Of All Time" : "For Now",
 		"Of All Time" : "Of The Last 30 Seconds",
-		"of all-time" : "for now",
 		"Of All-Time" : "For Now",
 		"OMG" : "*yawn*",
 		"OMG" : "No One Cares. At All",
 		"One Weird Trick" : "One Piece of Completely Anecdotal Horseshit",
 		"one weird trick" : "one piece of nonsensical folklore bullshit",
-		"priceless" : "painfully ordinary",
```

Comment:
The change in background.js make case-only differences redundant. It will match the upper-case version (and preserve) and because of the diversify method, it will ALSO match the lower-case and emit the lower-cased version of the replacement. Where there is already a _different_ mapping, it is left in place... this specific instance would be redundant.